Misc/NEWS.d/next/Core and Builtins/2021-09-26-18-18-50.bpo-45292.aX5HVr.rst

```diff
@@ -0,0 +1 @@
+Implement PEP 654: Add :class:`ExceptionGroup` and :class:`BaseExceptionGroup`.
```

Comment:
I don't have a plan, because I'm not sure how this works. Is the date in the filename meaningful or just for uniqueness?

Reply:
I think for sorting news items. People definitely update news items, occasionally. At release time all NEWS entries get bundled in file like Misc/NEWS.d/3.10.0b1.rst (newer ones are apparently only in the 3.10 branch), and it would be better if there was a single item for the initial implementation of EGs than having it spread across multiple dates.